keyboards/togkey/pad_pocket/keyboard.json

```
@@ -0,0 +1,51 @@
+{
+    "manufacturer": "TogKey",
+    "keyboard_name": "Pad Pocket",
+    "maintainer": "togkey86",
+    "development_board": "promicro_rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "rgblight": true,
+        "combo": true,
+        "tap_dance": true
```

These should be enabled only at the keymap level. This would currently not compile on the QMK Configurator, or in another keymap that lacks the necessary code for combos and tap dances.
```suggestion
        "rgblight": true
```